newIDE/app/src/UI/Theme/index.js

```diff
@@ -39,7 +55,7 @@ export function getTheme({
     console.warn(
       `Theme '${themeName}' is unavailable; '${defaultThemeName}' is used`
     );
-    theme = themes[defaultThemeName];
+    theme = themes[systemDefaultThemeName];
```

Comment:
I'm not this this will work, as you changed here the "fallback" theme which is the theme to use in case the `themeName` is invalid and can not be find in the `themes` map.

Basically, this is an error case. It should never happen. It can happen if you upgrade to a new version of GDevelop with a new theme, use this theme, then downgrade to a version without this theme (or if a theme is removed from GDevelop).

The theme is chosen from the preferences as you can see here: 
https://github.com/4ian/GDevelop/blob/0c22c52a783ff9fb665f1b617bf2a885afde2603/newIDE/app/src/MainFrame/Providers.js#L67-L72

This means that if you want to modify the default theme, it's something to do in the preferences. Where are the preferences stored when the user is launching GDevelop for the first time? 
From there: https://github.com/4ian/GDevelop/blob/0c22c52a783ff9fb665f1b617bf2a885afde2603/newIDE/app/src/MainFrame/Preferences/PreferencesContext.js#L216-L237
Precisely, the logic is in PreferencesProvider: 

https://github.com/4ian/GDevelop/blob/0c22c52a783ff9fb665f1b617bf2a885afde2603/newIDE/app/src/MainFrame/Preferences/PreferencesProvider.js#L34

I think that's here, or around these preferences files, that we should add the logic to choose the default theme :)

Reply:
Yes, you are right, that was a silly mistake.